flink-core/src/main/java/org/apache/flink/configuration/CleanupOptions.java

```diff
@@ -0,0 +1,195 @@
+
+    public static final String FIXED_DELAY_LABEL = "fixed-delay";
+    public static final String EXPONENTIAL_DELAY_LABEL = "exponential-delay";
+
+    public static final Set<String> NONE_PARAM_VALUES = ImmutableSet.of("none", "disable", "off");
+
+    private static String createParameterPrefix(String paramGroupKey) {
+        return CLEANUP_STRATEGY_PARAM + "." + paramGroupKey + ".";
+    }
+
+    private static String createFixedDelayParameterPrefix(String parameter) {
+        return createParameterPrefix(FIXED_DELAY_LABEL) + parameter;
+    }
+
+    private static String createExponentialBackoffParameterPrefix(String parameter) {
+        return createParameterPrefix(EXPONENTIAL_DELAY_LABEL) + parameter;
+    }
+
+    public static String extractAlphaNumericCharacters(String paramName) {
+        return paramName.replaceAll("[^a-zA-Z0-9]", "");
+    }
+
+    public static final ConfigOption<String> CLEANUP_STRATEGY =
+            ConfigOptions.key(CLEANUP_STRATEGY_PARAM)
+                    .stringType()
+                    .defaultValue(EXPONENTIAL_DELAY_LABEL)
+                    .withDescription(
+                            Description.builder()
+                                    .text(
+                                            "Defines the cleanup strategy to use in case of cleanup failures.")
+                                    .linebreak()
+                                    .text("Accepted values are:")
+                                    .list(
+                                            text(
+                                                    NONE_PARAM_VALUES.stream()
+                                                                    .map(ignored -> "%s")
+                                                                    .collect(
+                                                                            Collectors.joining(
+                                                                                    ", "))
+                                                            + ": Cleanup is only performed once. No retry "
+                                                            + "will be initiated in case of failure.",
+                                                    NONE_PARAM_VALUES.stream()
+                                                            .map(TextElement::code)
+                                                            .collect(Collectors.toList())
+                                                            .toArray(
+                                                                    new TextElement
+                                                                            [NONE_PARAM_VALUES
+                                                                                    .size()])),
+                                            text(
+                                                    "%s, %s: Cleanups will run in a static interval up "
```

Comment:
I think maybe something like "Cleanup attempts will be separated by a static interval/delay" would be a little clearer here, since "run in a static interval" sounds a little ambigious as to what its actually setting

Reply:
thanks for the suggestion. I like it 👍